chromium_src/components/translate/core/browser/brave_translate_manager_unittest.cc

```diff
@@ -0,0 +1,220 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
not sure if we should keep this in `chromium_src` now...

Reply:
I couldn't decide to keep this test here. So, I just copied the needed classes for CanManuallyTranslate_WithoutAPIKey and CanManuallyTranslate_WithAPIKey in order to avoid touching the upstream file for now.

Reply:
why did you change the name? There is no BraveTranslateManager class that I can see so this doesn't follow standard conventions

Reply:
IIRC, I renamed the class to fix a build break in the initial PR. But, the build error doesn't happen after moving test infra classes like TranslateManagerTest. ok, let me restore the name again.